content/en/docs/concepts/extend-kubernetes/_index.md

```diff
@@ -213,33 +247,44 @@ In addition to the built-in steps, there are several extensions:
   restricts what images can be run in containers.
 * To make arbitrary admission control decisions, a general
   [Admission webhook](/docs/reference/access-authn-authz/extensible-admission-controllers/#admission-webhooks)
-  can be used. Admission Webhooks can reject creations or updates.
+  can be used. Admission webhooks can reject creations or updates.
+  Some admission webhooks modify the incoming request data before it is handled further by Kubernetes.
 
-## Infrastructure Extensions
+## Infrastructure extensions
 
-### Storage Plugins
+### Device plugins
 
-[Flex Volumes](https://git.k8s.io/design-proposals-archive/storage/flexvolume-deployment.md)
-allow users to mount volume types without built-in support by having the kubelet call a binary
-plugin to mount the volume.
+_Device plugins_ allow a node to discover new Node resources (in addition to the
```

Comment:
In addition to the built-in Node resources (cpu and memory), _Device plugins_ allow a node to discover new resources through a [Device Plugin](/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/).

Reply:
I'm not sure that the wording you suggest would be idiomatic @kbhawkey, and maybe it could be misunderstood.

How about:
- Kubernetes is aware of built-in node-level system resources, such as `cpu` and `memory`.
  Kubernetes [device plugins](https://github.com/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/) allow a node to discover kinds of resources and make those available to the cluster, so that the scheduler can allocate those resources to Pods.

Reply:
Sure, I think I crossed paths when editing. Your first sentence looks good. The second sentence could be shortened (...so that the ...).

Reply:
From earlier in the file:

You can use [Device Plugins](#device-plugins) to integrate custom hardware or other special
   node-local facilities, and make these available to Pods running in your cluster. The kubelet
   includes support for working with device plugins.

And I think I read in one of the other pages in this PR (compute-storage-net):

Device plugins allow a node to discover new Node facilities (in addition to the
  built-in node resources such as `cpu` and `memory`), and provide these custom node-local
  facilities to Pods that request them.

Reply:
Sounds like it might be useful to update the glossary and make the working consistent.